controllers/conversion.go

```diff
@@ -178,11 +232,12 @@ func generateCfp(t target, patchCtx istio.EnvoyFilter_PatchContext, vhost *istio
 			},
 		},
 	}
-	if p.Name == util.Envoy_Ratelimit || p.Name == util.Envoy_Cors {
-		cfp.Patch = translateRatelimitToPatch(m.Inline.Settings, t.applyTo == istio.EnvoyFilter_HTTP_ROUTE)
-	} else {
-		cfp.Patch = translatePluginToPatch(p.Name, p.TypeUrl, m.Inline.Settings)
+
+	translator := patchTranslators[p.Name]
```

Comment:
We can make a more ideographic name. 

Actually when name matches ratelimit/cors, we patch to virtualhost/route itself. Otherwise we'll apply a plugin patch.